File: Exercícios/Lua/GestãoDeFuncionario/ClassUser.lua
-- Cadastros Gerais --
do --:Class_table

    -- Usuário | main -- ->>
    local Usuario = {} Usuario.__index = Usuario -- Table:Class

    function Usuario.nv(caract) --:Class_table =>
        local itc = setmetatable({}, Usuario)
        -- ->> --------------- <<- --
        itc.nome = caract.nome or 'None' --:string
        itc.tb_horas = caract.tb_horas or 0 --:float
        itc.dependentes = caract.dependentes or 0 --:int

        return itc -- <<=
    end -- <<-

    return Usuario --:Tables
end

File: Exercícios/Python/GestãoDeFuncionario/ClassUser.py
# -- Usuário | main -- ->>
class Usuário(object):
    """docstring for Usuario."""

    def __init__(self, user: dict):
        self.nome: str = user['nome']
        self.tb_horas: float = user['tb_horas']
        self.dependentes: int = user['dependentes']

    def print(self):
        print(f'{self.nome}\n{self.tb_horas}\n{self.dependentes}\n')
# -- <<-
